src/ucp/core/ucp_request.c

```diff
@@ -420,7 +420,6 @@ void ucp_request_send_state_ff(ucp_request_t *req, ucs_status_t status)
     ucp_request_id_release(req);
 
     if (req->send.uct.func == ucp_proto_progress_am_single) {
-        ucs_assert(req->send.state.uct_comp.count == 0);
```

Comment:
what is wrong with this assert?

Reply:
it wasn't set in `ucp_request_send_state_reset()` for AM_BCOPY protocols.
so, we shouldn't check it